README.md

```diff
@@ -63,7 +62,7 @@ With:
  - **-x** to deactivate authentication to the LDAP server
  - **-H ldap://127.0.0.1:3004** the server URL
  - **-b "dc=test"** the search base in LDAP directory, it should be the same as the **searchBase** property in server configuration above
- - **"(&(objectclass=person)(cn=user-login))"** the search filter, it should be the same as the **searchFilter** property in server configuration with **{{username}}** replaced by the user login
+ - **"(&(objectclass=person)(cn=user-login))"** the search filter
```

Comment:
An attribute objectClass with value "person" can also be added to the user-login user of the database for the ldapsearch command example to keep working.

Reply:
Sorry, I'm not entirely sure what you mean. Are you suggesting automatically adding that attribute to users in the database or are you just suggesting a change to the README?

Reply:
Apologies. I wasn't clear enough.
I meant just modifying the README to add objectClass attribute to the database user example in "LDAP users" section: 

```
[
  {
    "objectClass": "person",
    "dn": "cn=user,dc=test", // A valid DN (Distinguished Name)
    "cn": "user-login",
    "attribute1": "value1",
    "attribute2": "value2",
    [...]
  }
]
```

instead of:

```
[
  {
    "dn": "cn=user,dc=test", // A valid DN (Distinguished Name)
    "cn": "user-login", // The attribute corresponding to server configuration "userLoginAttribute"
    "attribute1": "value1",
    "attribute2": "value2",
    [...]
  }
]
```

Reply:
Ah, yes. Will do.

Reply:
Thank you!